pkgs/development/libraries/vigra/default.nix

```diff
@@ -69,14 +66,18 @@ stdenv.mkDerivation (finalAttrs: {
         doCheck = stdenv.buildPlatform.canExecute stdenv.hostPlatform;
       });
     };
+    updateScript = writeShellScript "update-vigra" ''
+      latestVersion=$(curl ''${GITHUB_TOKEN:+-u ":$GITHUB_TOKEN"} --fail --silent https://api.github.com/repos/ukoethe/vigra/releases/latest | ${lib.getExe jq} --raw-output .tag_name | sed -E 's/Version-([0-9]+)-([0-9]+)-([0-9]+)/\1.\2.\3/')
+      ${lib.getExe nix-update} vigra --version $latestVersion
+    '';
```

Comment:
How about splitting the `updateScript` part into its own commit before the version bump? That would make it easier to test.

BTW, how does it get the GITHUB_TOKEN?

Reply:
Ah, I see. We don't run update scripts in a Nix sandbox.